packages/material-ui-unstyled/src/composeClasses/composeClasses.ts

```diff
@@ -1,5 +1,5 @@
 export default function composeClasses<ClassKey extends string>(
-  slots: Record<ClassKey, ReadonlyArray<string | false>>,
+  slots: Record<ClassKey, ReadonlyArray<string | false | undefined>>,
```

Comment:
Why was `undefined` needed?

Reply:
If I don't add `undefined`, then I got error on this.
![image](https://user-images.githubusercontent.com/18292247/115435753-61ef3d80-a234-11eb-8be0-820425daee2e.png)

and I think adding undefined to the type make sense because the implementation also handle it.
```ts
Object.keys(slots).forEach(
    (slot: ClassKey) => {
      output[slot] = slots[slot]
        .reduce((acc, key) => {
          if (key) { // here key can be undefined and does not affect the implementation
            if (classes && classes[key]) {
              acc.push(classes[key]);
            }
            acc.push(getUtilityClass(key));
          }
          return acc;
        }, [] as string[])
        .join(' ');
    },
  );
```

Reply:
ah correct for the keys that are dependent on props. Makes sense!